File: RepairYourPCForEmployee/app/src/main/java/kingapawlowska/com/repairyourpcforemployee/Order.java
package kingapawlowska.com.repairyourpcforemployee;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

/**
 * Created by Kinga on 2017-01-18.
 */
public class Order implements Serializable {

    @SerializedName("id")
    public int id;

    @SerializedName("login")
    public String login;

    @SerializedName("repairIdentifier")
    public String repairIdentifier;

    @SerializedName("manufacturer")
    public String manufacturer;

    @SerializedName("kindOfHardware")
    public String kindOfHardware;

    @SerializedName("model")
    public String model;

    @SerializedName("service")
    public String service;

    @SerializedName("operatingSystem")
    public String operatingSystem;

    @SerializedName("description")
    public String description;

    @SerializedName("link")
    public String link;

    @SerializedName("repairStatus")
    public String repairStatus;

    @SerializedName("estimatedCost")
    public String estimatedCost;

    @SerializedName("paid")
    public String paid;

    @SerializedName("created_at")
    public String created_at;

    @SerializedName("updated_at")
    public String updated_at;
}

File: repair_your_pc_php/include/order.php
<?php

require_once '../include/user.php';
$login = "";
$repairIdentifier = "";
$manufacturer = "";
$kindOfHardware = "";
$model = "";
$service = "";
$operatingSystem = "";
$description = "";
$link = "";

if(isset($_POST['login']))
{
    $login = $_POST['login'];
}

if(isset($_POST['repairIdentifier']))
{
    $repairIdentifier = $_POST['repairIdentifier'];
}

if(isset($_POST['manufacturer']))
{
    $manufacturer = $_POST['manufacturer'];
}

if(isset($_POST['kindOfHardware']))
{
    $kindOfHardware = $_POST['kindOfHardware'];
}

if(isset($_POST['model']))
{
    $model = $_POST['model'];
}

if(isset($_POST['service']))
{
    $service = $_POST['service'];
}

if(isset($_POST['operatingSystem']))
{
    $operatingSystem = $_POST['operatingSystem'];
}

if(isset($_POST['description']))
{
    $description = $_POST['description'];
}

if(isset($_POST['link']))
{
    $link = $_POST['link'];
}

// Instance of a User class
$userObject = new User();

// Registration of new order
if(!empty($login) && !empty($repairIdentifier) && !empty($kindOfHardware) && !empty($service) && !empty($operatingSystem) && !empty($description))
{
    $json_registration_order = $userObject->createNewRegisterOrder($login, $repairIdentifier, $manufacturer, $kindOfHardware, $model, $service, $operatingSystem, $description, $link);
    echo json_encode($json_registration_order);
}

?>